element_calcium_imaging/scan.py

```diff
@@ -687,6 +687,12 @@ def make(self, key):
             )
 
             quantalsize_results = compute_quantal_size(scan)
+            middle_frame = scan.shape[0] // 2
+            quantal_frame = (
+                np.mean(scan[max(middle_frame - 250, 0) : middle_frame + 250], axis=0)
```

Comment:
250 is a magic number. Perhaps you could assign it to a named constant.

Do you know that `middle_frame >= 250`?

Reply:
How about this?

```python
HALF_SHIFT = min(HALF_SHIFT, middle_frame)
scan[max(middle_frame - HALF_SHIFT) : middle_frame + HALF_SHIFT]
```

Reply:
great